ios/Classes/Bridging/CharacteristicContainer.h

```diff
@@ -0,0 +1,15 @@
+#import "Characteristic.h"
+#import "Descriptor.h"
+
+@interface CharacteristicContainer : NSObject
+
+@property (readonly) Characteristic *characteristic;
+@property (readonly) NSArray<Descriptor *> *descriptors;
+
+- (instancetype)initWithCharacteristic:(Characteristic *)characteristic
+                           descriptors:(NSArray<Descriptor *> *)descriptors;
+
+- (NSArray *)descriptorsJsonRepresentationForCharacteristic;
```

Comment:
I'd say that name of this method could be shortened to `descriptorsJsonRepresentation`.